app/index.html

```diff
@@ -1,4 +1,4 @@
-<!doctype html>
```

Comment:
We have >300 occurences of  `<!doctype` and <100 occurences of `<!DOCTPYE` let's stick with the lower-cased variant. Does this break IE?